templates/terraform/custom_expand/computed_subscription_topic.erb

```diff
@@ -27,7 +27,9 @@ func expand<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d T
         } else {
 		// If no full topic given, we expand it to a full topic on the same project
 		fullTopic := fmt.Sprintf("projects/%s/topics/%s", project, topic)
-		d.Set("topic", fullTopic)
+		if err := d.Set("topic", fullTopic); err != nil {
```

Comment:
Oh no! The SDK path change meant that the rule excluding `d.Set` from the `errcheck` lint broke: https://github.com/hashicorp/terraform-provider-google/blob/master/.golangci.yml#L26

This is an improvement to the code so I'd imagine it's worth keeping now that you've made it- sorry that you had to, though.

Reply:
oooooooh, that's what did it!  oh no :( I was so confused why it was just showing up now, but now that I see it there it makes sense.